CHANGELOG.md

```diff
@@ -1,30 +1,41 @@
 ## 16.0.0
 
+## 16.0.1
```

Comment:
This seems a bit weird that it added `16.0.1` after `16.0.0`. We might can see what happens with `16.0.2`.

Reply:
oh yeah, might need to manually move that